src/nvim/diff.c

```diff
@@ -2017,17 +2017,24 @@ int diff_infold(win_T *wp, linenr_T lnum)
 }
 
 /// "dp" and "do" commands.
-///
-/// @param put
-void nv_diffgetput(int put)
+void nv_diffgetput(bool put, size_t count)
 {
   exarg_T ea;
-  ea.arg = (char_u *)"";
+  char buf[30];
+
+  if (count == 0) {
+    ea.arg = (char_u *)"";
+  } else {
+    vim_snprintf(buf, 30, "%ld", count);
```

Comment:
To answer your question @justinmk, I tested with this code:

```c
#include <stdio.h>
#include <BaseTsd.h>

int main()
{
	size_t x = 1;
	SSIZE_T y = 2;
	printf("%zu\n", x);  // prints as unsigned decimal
	printf("%zx\n", x);  // prints as hex
	printf("%zd\n", y);  // prints as signed decimal
	return 0;
}
```

on both VS2013 and VS2015.

VS2015 outputs:

```
1
1
2
```

while VS2013 says:

```
zu
zx
zd
```

(note `ssize_t` is a POSIX extension and `SSIZE_T` is similar thing in [Windows Type](http://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx), hence I added `<BaseTsd.h>`)

Let me know if there is anything else you want me to confirm.